### DISCUSSION TITLE
YOLO badge is a bit unfair

### DISCUSSION THREAD

**Discussion by pasmontesinos** [community/community#19847] [Other Feature Feedback, Questions, & Ideas]
I think the YOLO badge has a negative connotation. For example, in my team, we use the [Ship Show Ask strategy](https://martinfowler.com/articles/ship-show-ask.html), and is common to merge without revision when the PR is labeled as "Ship" or "Show". I think is unfair to get that badge

**Comment by sinkaroid** [comment]
The YOLO names tell it all, **"You want it? You merge it."**, In my opinion yolo badge just a gimmick
